src/apollo/global/globalTypes.json

```diff
@@ -1,17 +1,25 @@
 {
 	"operations": [
```

Comment:
Should we add this file to `.gitignore`?

Reply:
If i remember correctly this file is automaticaly generated when you pull schema from the server. If you add it to git ignore, then you have to pull from server everytime you are missing this file.